### DISCUSSION TITLE
"Code 37 is not supported" Error when reading NEXRAD Level 3 files

### DISCUSSION THREAD

**Discussion by turbulentlens** [ARM-DOE/pyart#1358] [Q&A] [answered]
Hello,

I'm getting this odd error when trying to read untarred NEXRAD L3 files acquired through the NCEI Archive. Any help with the workflow of getting archived NEXRAD L3 files read into Pyart would be really appreciated. I've attempted to use NOAA WCT to import netCDFs of the NEXRAD files but I don't believe they're formatted compatibly with pyart. My end goal is to plot the KHGX radar with the TRACER C-Band. 

<img width="1171" alt="Screenshot 2022-12-16 at 1 27 23 PM" src="https://user-images.githubusercontent.com/61855795/208165015-58898f7e-9de0-46b3-8f6a-c473953386e9.png">

**Comment by zssherman** [comment]
Hi @jpena9! I am not familiar with message code 37, I'll discuss with @mgrover1 on this to see what the error entails.

**Comment by jrwebster** [comment, marked as answer]
Hi @jpena9,

I believe the error results from trying to read an unsupported product (due
to the file not being in a radial image message format). See line 866 of
pyart/io/nexrad_level3.py

Best,

Jason

**Reply by zssherman** [reply]
Ah! Yep good catch! Thanks @jrwebster

**Reply by turbulentlens** [reply]
Ah I see what you mean now! Thank you both for the help.
